### ISSUE TITLE
Option to force frontend reload from Minecraft based on version

### ISSUE THREAD

**Issue by creesch** [creesch/minecraft-web-chat#21]
Basically let the websocket interface start with sending the mod version on connect. If the webinterface receives a different version than expected force a reload of the page.

This should prevent any issues with people updating the mod while leaving chat open in the browser.

**Comment by creesch** [comment]
To avoid adding the version to static files probably leave it as an empty variable first. Then the first connect sets the version as that is a fresh page load. Then on reconnect we check against the stored version.

**Comment by creesch** [comment]
Depends on #22

**Comment by danthedaniel** [comment]
Make sure assets aren't cached as well.

**Comment by creesch** [comment]
1. On the server end I was thinking of exposing a dynamic endpoint `/mod-version` which just returns the version of the mod (maybe do it nicely in mod format). 
2. On page load the client loads that resource and uses it to set a constant. 
3. WebsocketJsonMessage needs to be updated to include the mod version.
4. If the client then receives a message with a different mod version than the constant it means the user left the tab open while the mod was updated. 
5. Reload page. 


I am fairly sure I saw a Fabric API call to get the version of the mod.  Doing it like this prevents us from having to store the version in multiple places.

**Comment by creesch** [comment]
Google says 

```
        String version = FabricLoader.getInstance()
            .getModContainer("web-chat")
            .get()
            .getMetadata()
            .getVersion()
            .getFriendlyString();
```

https://maven.fabricmc.net/docs/fabric-loader-0.16.9/net/fabricmc/loader/api/FabricLoader.html

getModContainer is an optional.

**Comment by danthedaniel** [comment]
I think I'd do this without adding another endpoint for the mod version. Just store the version from the first message over the websocket. Also - this should be the default behavior and I see no reason for it to be an option. Are you okay with me tackling this one, @creesch?

**Comment by creesch** [comment]
> Just store the version from the first message over the websocket.

I don't think we are sending the mod version to the front end at this moment. So that would need to be added, other than that I am okay with you picking it up.
